src/renderer/components/PrivateKeyField/PrivateKeyField.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from "react";
+import {InputForm, IInputFormProps} from "../Input/InputForm";
+import {useState, useEffect} from "react";
+import {PasswordPrompt} from "../Prompt/PasswordPrompt";
+import {ISubmitStatus} from "../Prompt/InputPrompt";
+
+export interface IPrivateKeyProps extends IInputFormProps {
+    password: string;
+}
+export const PrivateKeyField: React.FunctionComponent<IPrivateKeyProps> = (props: IPrivateKeyProps) => {
+    const[showPrompt,setShowPrompt]=useState<boolean>(false);
+    const[privateKey/*, setPrivateKey **/]=useState<string | undefined>(props.inputValue);
+    const[timeoutStatus, setTimeoutStatus]=useState<boolean>(false);
+    const[passwordType, setPasswordType]=useState<string>("password");
```

Comment:
if there can be only two types - password and text, is it better to use a boolean flag like `hidden` ?